src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -11,16 +31,60 @@ schedules:
 trigger:
   batch: true
   branches:
-   include:
-   - release/*
-   - internal/release/*
+    include:
+    - main

```

Comment:
Correct, thanks for reminding me about this. I wanted to test what happens if I move the `stages` section under `if not(variables.disableBuild)`, I _think_ it should show up as a skipped build then.

Reply:
@akoeplinger fwiw, you can add this for no-op stages:

```yml
    jobs:
      - job: No_op
        pool: server
```

Reply:
@premun thanks, yeah the only problem with that is that the build would show up as passing even though it was a no-op which I think is what @MichaelSimons wanted to avoid

Reply:
Correct - A build green build run should not appear when it no-ops, that would give a false impression and create noise.

Reply:
I tried a few things but nothing works the way we want.

I think the easiest way would be to remove the triggers from ci.yml completely and configure them through the AzDO UI, that way we can have different triggers per pipeline. Any objections?

Reply:
>  I think it would be helpful to have a yml comment indicating that triggers exist in the UI.

Yes, definitely.

Reply:
Ok, pushed.